modules/social_features/social_event/src/Entity/EventEnrollment.php

```diff
@@ -110,10 +110,19 @@ public function getName() {
    * {@inheritdoc}
    */
   public function label() {
+    // When the guest is allowed to enjoin, the name and account will be empty,
+    // but it will have field_email, the first and last name isn't mandatory,
+    // because that the field_name is being used to validate.
```

Comment:
I find this why explanation a bit clumsy. Do you think we can improve it?

I suggest something clearer but if you have another idea feel free to do so.
`When a guest is allowed to join, the name and account fields will be empty, but the field for email will be provided. The first and last name are not mandatory, so the field_name is used for validation instead.`